cli/src/main/java/org/eclipse/hono/cli/Receiver.java

```diff
@@ -10,80 +10,90 @@
  *    Bosch Software Innovations GmbH - initial creation
  *
  */
-package org.eclipse.hono.example;
+package org.eclipse.hono.cli;
 
 import javax.annotation.PostConstruct;
 
+import io.vertx.core.AsyncResult;
+import io.vertx.core.CompositeFuture;
 import io.vertx.core.Handler;
 import org.apache.qpid.proton.amqp.messaging.AmqpValue;
 import org.apache.qpid.proton.amqp.messaging.Data;
 import org.apache.qpid.proton.amqp.messaging.Section;
 import org.apache.qpid.proton.message.Message;
 import org.eclipse.hono.client.HonoClient;
-import org.eclipse.hono.client.MessageConsumer;
 import org.eclipse.hono.util.MessageHelper;
 import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Component;
 
 import io.vertx.core.Future;
 import io.vertx.proton.ProtonConnection;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
- * Example of a event/telemetry receiver that connects to the Hono Server, waits for incoming messages and logs the message
- * payload if anything is received.
+ * Receiver that connects to the Hono Server, waits for incoming messages and logs the message payload if anything is
```

Comment:
In fact, there hasn't been a *Hono Server* component anymore for quite some time now. Let's call it simply *Hono* ...